releasenotes.txt

```diff
@@ -1,4 +1,27 @@
-﻿Build 5.0.0
+﻿Build 5.0.1
+=============================
+
+Release notes - NHibernate - Version 5.0.1
+
+** Bug
+    * #1419 ISession.IsDirty() shouldn't throw exception for transient many-to-one object in a session
```

Comment:
Manually reformated this way from something like
```
__Bug__

[__#1419__](https://url) ...
```
In which PR were listed too, which I removed, identifying them thanks to their url.